### PR TITLE
bc: add Japanese translation

### PR DESCRIPTION
**Pull request by t-mangoe**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ja/common/bc.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+`bc --mathlib`
+
+- 式を計算する:
+
+`echo '{{5 / 3}}' | bc`
+
+- スクリプトを実行する:
+
+`bc {{path/to/script.bc}}`
```
Comment (kbdharun) [inline pages.ja/common/bc.md:21]: Thanks for your contribution, can you translate this path placeholder too?

Comment (t-mangoe) [reply pages.ja/common/bc.md]: Thanks for your comment. I tried to translate it.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.ja/common/bc.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# bc
+
+> 任意の精度で計算を行える言語です。
+> `dc`も参照してください。
+> 詳細はこちら: <https://manned.org/man/bc.1>.
```
Comment (blueskyson) [inline pages.ja/common/bc.md:5]: We have a [translation](https://github.com/tldr-pages/tldr/blob/a36f018973cffa325638e4131f8ceb0336f019df/scripts/set-more-info-link.py#L25) sheet for more info. 
```suggestion
> 詳しくはこちら: <https://manned.org/man/bc.1>.
```

**Review by blueskyson — APPROVED** [review]

**Pull request merged**